application_form/target.md

```diff
@@ -0,0 +1,57 @@
+- Being open to people of any gender
+
+## A brief introduction to gender
+
+A lot of people are assigned a gender at birth, and continue to identify with that gender - they feel fine about it. These people are called cisgender (or cis women/men). Some people are transgender, which means they do not identify with that gender. Most of these people have been assigned female at birth (AFAB) or assigned male at birth (AMAB), .
+
+There is a wide variety amongst trans people. Some identify entirely as a man or woman, some feel like they belong somewhere in between. The latter group is often called non-binary, but different labels exist. A lot of trans people don't conform to their assigned gender in appearance (their gender expression). They may use a different name, pronouns or change (some of) their body with medical processes. Not all of them do, however. People who have transitioned from man to woman are called trans women, people who transition from woman to man are called trans men.
+
+Many trans and non-binary people encounter difficulties in life, and in entering technology, that are not encountered by cisgender people - especially compared to cis men. Therefore, it can make sense to open your workshop to this wider group.
+
+## Being open to women
+
+Opening up applications to women, including trans women, is a common choice for Django Girls events. The website defaults to this, but we recommend adding an explicit mention of being trans inclusive. A good way to word this is:
+
+> We are trans-inclusive and welcome applications from all women.
+
+## Being open to women and non-binary people
+
+Another choice is to be open to women and non-binary people. Non-binary people can be AMAB or AFAB, and can have any range of appearances. Some might present in a way that others will read as them being men or women, some look more ambiguous. This is influenced by personal preference, but also by safety concerns. Therefore, you need to be aware that some attendees may identify as non-binary, yet not look different to you than cis men or cis women.
+
+The Django Girls name and website refer to "women" and "girls" in a few places already, so if you use the phrase "women and non-binary people" on your event page, people are likely to assume that you mean "non-binary people who look like women to the organisers" or "people who are feminine enough in their appearance" or "only AFAB non-binary people". To avoid this, it's important to be very explicit.
+
+A good start for being and appearing inclusive is:
+
+- Early on in your event page, clarify your target group as: “women and non-binary people (regardless of gender expression or assigned gender at birth)”
+- Consistently using “women and non-binary people” everywhere, and never “women” alone.
+- Ask for people’s pronouns in the application form - this target group means you can’t assume "she" as the default pronoun. If you make name badges, try to print the pronouns on them.
```

Comment:
I have mixed feelings about printing pronouns on name badges (or forcing everybody to do it). I write my pronouns on badges, and know other people who do.

But it’s very unusual to have my pronouns confused, so the risk to me is minimal. And people who are openly trans/non-binary do the same – the same pronouns you might see on Twitter. I wonder if everybody is comfortable doing this?

e.g. if you’re non-binary but aren’t out to everyone, do you put they/them pronouns (thus outing yourself), or put pronouns you’re not comfortable with?

Disclaimer: cis man who currently has no issues with the pronouns people who use for him, so I may be talking nonsense.

Reply:
I've added the suggestion to make this question optional.